astroquery/mast/missions.py

```diff
@@ -77,17 +85,24 @@ def query_region_async(self, coordinates, radius=3*u.arcmin, **kwargs):
             The string must be parsable by `~astropy.coordinates.Angle`. The
             appropriate `~astropy.units.Quantity` object from
             `~astropy.units` may also be used. Defaults to 3 arcminutes.
+        limit : int
+            the maximun number of dataset IDs in the results.
+        offset : int
+            the number of records you wish to skip before selecting records.
         **kwargs
             Other mission-specific keyword args.
-            These can be found at the following link
-            https://mast.stsci.edu/search/docs/#/Hubble%20Search/post_search_hst_api_v0_1_search_post
+            Any invalid keys are igonerd by the API.
+            All valid key names can be found using `~astroquery.mast.missions.MastMissionsClass.get_column_list`
+            function.
             For example one can specify the output columns(select_cols) or use other filters(conditions)
```

Comment:
I'm not sure I understand the wording here, is this saying that there are two key names called `select_cols` and `conditions`?